test/autoCompletion.test.ts

```diff
@@ -786,7 +786,7 @@ describe('Auto Completion Tests', () => {
       });
 
       it('Insert required attributes at correct level', (done) => {
-        const schema = require(path.join(__dirname, './fixtures/testRequiredProperties.json'));
+        const schema = require('./fixtures/testRequiredProperties.json');
```

Comment:
Apparently this trades one code scanning alert for another, though the new one is not as bad as the old one.

I do agree with the new failed checks. The proper solution would be to write this at the top of the file:

```ts
import schema = require('./fixtures/testRequiredProperties.json');
```